pkg/restclient/common.go

```diff
@@ -0,0 +1,73 @@
+package restclient
```

Comment:
Instead of calling this package, you can just call it `gd2client` or just `client`. We don't expect there to be any other type of GD2 client package. 
This will make more sense once we restructure the source tree, then this package would appear as `glusterd2/rest/client`.